src/Popover/Popover.js

```diff
@@ -112,7 +112,7 @@ class Popover extends Component {
   constructor(props, context) {
     super(props, context);
     this.handleResize = throttle(this.setPlacement, 100);
-    this.handleScroll = throttle(this.setPlacement.bind(this, true), 100);
+    this.handleScroll = throttle(this.setPlacement.bind(this, true), 50);
```

Comment:
I have tested with a `x20` CPU throttling, that seems reasonable to lower the level.